### DISCUSSION TITLE
Manual Installation

### DISCUSSION THREAD

**Discussion by Aymkdn** [valinet/ExplorerPatcher#2644] [Ideas]
Hello,

Would it be possible to have a way to manually install Explorer Patcher instead of using the `.exe` setup file? See the reasons [there](https://github.com/valinet/ExplorerPatcher/discussions/2639).

What I imagine:
1. `ep_setup.exe /extract` will extract the same files as of today, but also the `dxgi.dll` and `ep_gui.dll` files.
2. A wiki page to explain that `dxgi.dll` should be placed in `C:\Windows\` and the other files in `C:\Program Files\ExplorerPatcher`, and then restart Windows Explorer.

Thanks.

**Comment by pyrates999** [comment]
The automatic install method is required because that is used to update it when a new release is out.

**Reply by Aymkdn** [reply]
The idea is to also manually update the files when a new release is out. In my case, the executable setup is blocked by my computer and I cannot install it.
Furthermore, I didn't receive any updates yet so I don't know what it will happen.

Hopefully, doing the manual install worked :-)

**Reply by pyrates999** [reply]
Not without it automatically downloading symbols for your release version of windows.  And they need to be redownloaded every month.

If the exe is blocked on your computer, there's a reason for that.  I would not be installing this on any work computer since the admin privilege's can be taken away at anytime.

**Reply by Aymkdn** [reply]
BTW the symbols downloaded without issues.

**Reply by pyrates999** [reply]
do you get notifications as well?

**Reply by Aymkdn** [reply]
> do you get notifications as well?

Notifications of what? Notifications about new version? I don't know yet as I've just installed it.

**Reply by pyrates999** [reply]
you should get notifications for symbols starting to be downloaded, symbols have downloaded, and if there is a new version or not when you check for a new version from the EP properties application.

**Comment by Aymkdn** [comment]
> you should get notifications for symbols starting to be downloaded, symbols have downloaded, and if there is a new version or not when you check for a new version from the EP properties application.

Yes, I'm getting all the notifications.
